geo/src/main/scala/io/idml/geo/CityFunction.scala

```diff
@@ -36,8 +39,8 @@ class CityFunction(driver: String, url: String, user: String, password: String)
       .transact(xa)
       .attempt
       .unsafeRunSync()
-      .leftMap { _ =>
-        // could log the error here
+      .leftMap { e =>
+        log.warn("admin1 couldn't be called", e)
```

Comment:
This should be `city couldn't be called`

Reply:
good point, fixed